### PR TITLE
Fix #270 (CI) - file license changes not showing

### PR DESCRIPTION
**Pull request by gianpaj**
This fixes issue: https://github.com/clearlydefined/website/issues/270

### CODE REVIEW

File: src/utils/contribution.js [hunk 1]
```diff
@@ -81,12 +81,16 @@ export default class Contribution {
...
   /**
+   *
    * Applies changes to the specific component
+   * @static
    * @param  {} component original component
    * @param  {} changes object containing changes
    * @param  {} field field to check
    * @param  {} value value to apply to the field
    * @param  {} type used to specify a type of field (e.g. array) to perform different kind of operations
+   * @param {*} [transform=a => a]
```
Comment (AlexWebYourmind) [inline src/utils/contribution.js]: What is this new @param {*} for? 
Make sure to add also a comment specifying its usage (in the code)

File: src/utils/contribution.js [hunk 2]
```diff
@@ -81,12 +81,16 @@ export default class Contribution {
...
+   * @returns
```
Comment (AlexWebYourmind) [inline src/utils/contribution.js]: @gianpaj can you confirm you wanted this @returns doc comment?

**Comment by gianpaj** [comment]
@AlexWebYourmind I've amended the documentation based on your feedback

File: src/utils/contribution.js (hunk 2 above)
Comment (gianpaj) [reply src/utils/contribution.js]: that's what JSdoc added automatically. I've removed it

File: src/utils/contribution.js (hunk 1 above)
Comment (gianpaj) [reply src/utils/contribution.js]: I'm not the correct convention. But I've replaced `{*}`with `{}` for consistency.

I've also added a comment as you correctly pointed out

**Pull request merged**